app/assets/stylesheets/components/_firm.scss

```diff
@@ -13,8 +13,10 @@
 }
 
 .firm__adviser-distance {
-  margin-bottom: $baseline-unit*2;
+  margin-left: -$baseline-unit/2;
```

Comment:
$baseline-unit is only a vertical/top/bottom concept and doesn't need to be used for horizontal positioning